### PR TITLE
Adding documentation of two new campaign related options

### PR DESCRIPTION
**Pull request by merltron**
Adding documentation of "campaign_check" and "campaign_accept" which
needs developer review.

I'm not entirely sure about my explanation for "accept". In the UI, there are some campaigns that the end-user (ie: motorist) has to accept before installing, and some campaigns that are allowed to auto install. Does this "accept" refer to the same thing?

Also, why is the "full" mode not documented anywhere?

### CODE REVIEW

File: docs/selectively-triggering-aktualizr.adoc [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 To run the update cycle manually, first stop and disable the aktualizr background service:
 
     systemctl stop aktualizr && systemctl disable aktualizr
 
-Assuming aktualizr is being built using Yocto and meta-updater, the systemd service can also be disabled by default by adding the following to `local.conf`:
+Assuming aktualizr is being built using Yocto and meta-updater, the `systemd` service can also be disabled by default by adding the following to `local.conf`:
 
-    SYSTEMD_AUTO_ENABLE_aktualizr = "disable"
+    `systemd`_AUTO_ENABLE_aktualizr = "disable"
```
Comment (patrickvacek) [inline docs/selectively-triggering-aktualizr.adoc]: The backticks are probably undesirable in this instance, unless you want to wrap all of the similar commands and configuration bits completely in backticks.

Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: ah sorry, that was a find and replace mistake. it shouldn't have been there.

Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: fixed.

File: docs/selectively-triggering-aktualizr.adoc [hunk 2]
```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
...
+
+    aktualizr --running-mode=campaign_check
+
+If the device matches one or more campaigns, aktualizr prints details of each applicable campaign including the campaign ID. You can then use the campaign ID for the `campaign_accept` command.
+
+=== Accept campaigns that target the device
+
+This running mode works together with the `campaign_check` mode. After you have received a list of applicable campaigns, you can accept specific campaigns so that they update the device.
+
+    aktualizr --running-mode=campaign_accept <CAMPAIGN_ID>
```
Comment (lbonn) [inline docs/selectively-triggering-aktualizr.adoc]: The format is actually:

    aktualizr --running-mode=campaign_accept --campaign-id=<CAMPAIGN_ID>

File: docs/selectively-triggering-aktualizr.adoc [hunk 3]
```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
 
     aktualizr --running-mode=check
 
+=== Check for campaigns target the device
+
+In the OTA Connect web app, you can create smart groups that filter for devices based on specific criteria. When OTA Connect detects a new device that matches this criteria, it automatically publishes the campaign to the device. This running mode checks to see if the device matches the filter criteria used in an active campaign.
```
Comment (lbonn) [inline docs/selectively-triggering-aktualizr.adoc]: Evoking smart groups seems a bit unnecessary. An user can create campaigns with static groups as well and I'd say it's a detail of the campaign creation process.

Maybe just a brief summary of what a campaign is in abstract (setup of an update to a fleet, with associated metadata) would suffice?

File: docs/selectively-triggering-aktualizr.adoc (hunk 2 above)
Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: Ok fixed

File: docs/selectively-triggering-aktualizr.adoc (hunk 3 above)
Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: Ok added a more general description.

File: docs/selectively-triggering-aktualizr.adoc [hunk 4]
```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
 
     aktualizr --running-mode=check
 
+=== Check for campaigns target the device
```
Comment (lbonn) [inline docs/selectively-triggering-aktualizr.adoc]: `that` is missing here

File: docs/selectively-triggering-aktualizr.adoc [hunk 5]
```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
...
+
+=== Accept campaigns that target the device
+
+This running mode works together with the `campaign_check` mode. After you have received a list of applicable campaigns, you can accept the campaigns so that they are included in the software update process.
+
+    aktualizr --running-mode=campaign_accept --campaign-id=<CAMPAIGN_ID>
+
+For example:
+
+    aktualizr --running-mode=campaign_accept --campaign-id=4df1a199-59d4-47f6-a261-d79b83020f65
```
Comment (lbonn) [inline docs/selectively-triggering-aktualizr.adoc]: I would add that the update scheduled by the campaign will then become available to the device (`check` running mode or regular full update cycle)

Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: Ok done, see latest commit

File: docs/selectively-triggering-aktualizr.adoc (hunk 4 above)
Comment (merltron) [reply docs/selectively-triggering-aktualizr.adoc]: Ok done, see latest commit

**Comment by merltron** [comment]
Added more fixes in latest commit. https://github.com/advancedtelematic/aktualizr/pull/939/commits/cf4ad2c0b9038ff715a113e9db94a549fd8af519

**Pull request merged**